JuiceFS 1.4.0-beta1 panics on Windows Server 2022 — all subcommands

<!--
Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
-->

**What happened**:

Running juicefs-1.4.0-beta1.exe on Windows Server 2022 panics immediately, regardless of subcommand. Even --version panics.

```
PS C:\> C:\juicefs\juicefs-1.4.0-beta1.exe status $env:JFS_META_URL
panic: Failed to find ldap_first_entryW procedure in wldap32.dll:
       The specified procedure could not be found.
        panic: Failed to find ldap_msgfreeW procedure in wldap32.dll:
               The specified procedure could not be found.

goroutine 1 [running]:
golang.org/x/sys/windows.(*LazyProc).mustFind(...)
        /go/pkg/mod/golang.org/x/sys@v0.42.0/windows/dll_windows.go:328
golang.org/x/sys/windows.(*LazyProc).Call(0x1438b6ac0, {0xc00000bfc8, 0x1, 0x1})
        /go/pkg/mod/golang.org/x/sys@v0.42.0/windows/dll_windows.go:350 +0x59
panic({0x143e1dca0?, 0xc0004a9d10?})
        /usr/local/go/src/runtime/panic.go:783 +0x132
golang.org/x/sys/windows.(*LazyProc).mustFind(...)
        /go/pkg/mod/golang.org/x/sys@v0.42.0/windows/dll_windows.go:328
golang.org/x/sys/windows.(*LazyProc).Call(0x1438b69c0, {0xc000548060, 0x2, 0x2})
        /go/pkg/mod/golang.org/x/sys@v0.42.0/windows/dll_windows.go:350 +0x59
github.com/juicedata/juicefs/pkg/win.LdapGetValue(0x477b0c48, ...)
        /go/src/github.com/juicedata/juicefs/pkg/win/ldap.go:123
github.com/juicedata/juicefs/pkg/win.LdapGetDefaultNamingContext(...)
        /go/src/github.com/juicedata/juicefs/pkg/win/ldap.go:142
github.com/juicedata/juicefs/pkg/win.initializeTrustPosixOffsets()
        /go/src/github.com/juicedata/juicefs/pkg/win/sid.go:183
github.com/juicedata/juicefs/pkg/win.init.0()
        /go/src/github.com/juicedata/juicefs/pkg/win/sid.go:313
```

**What you expected to happen**:

JuiceFS commands should execute without panicking.

**How to reproduce it (as minimally and precisely as possible)**:

1. Place juicefs-1.4.0-beta1.exe on Windows Server 2022
2. Run any subcommand, e.g. juicefs-1.4.0-beta1.exe --version
3. Observe the panic immediately -- no output is produced

**Anything else we need to know?**

Build commit: 0529bd2f57d504717f721eddbe4b6d1f98511291 (tag: "bump to release 1.4.0-beta1 (#6984)")
Binary SHA256: 7E42A5AD588FC4E3C33864D39D3673422FA7A672867EDF4F9CB35B45342797A6
Binary size: 127,399,936 bytes

wldap32.dll: 10.0.20348.2849, 64-bit, 393,216 bytes, C:\Windows\System32\wldap32.dll

**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version: v1.4.0-beta1 (crash prevents --version from producing output, confirmed via embedded binary strings)
- Cloud provider or hardware configuration running JuiceFS: N/A -- crash occurs before any subcommand logic
- OS (e.g `cat /etc/os-release`): Microsoft Windows Server 2022 Standard, version 10.0.20348, build 20348, 64-bit
- Kernel (e.g. `uname -a`): N/A (Windows)
- Object storage (cloud provider and region, or self maintained): N/A
- Metadata engine info (version, cloud provider managed or self maintained): N/A
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage): N/A
- Others: Crash traced to pkg/win/ldap.go lines 37/41 which use W-suffixed LDAP function names that do not exist in the Windows LDAP API. Occurs in Go init() before main() -- all subcommands affected.

--- a/pkg/win/ldap.go
+++ b/pkg/win/ldap.go
@@ -34,11 +34,11 @@ var (
 	procLdapBindSW       = modWldap32.NewProc("ldap_bind_sW")
 	procLdapUnbind       = modWldap32.NewProc("ldap_unbind")
 	procLdapSearchSW     = modWldap32.NewProc("ldap_search_sW")
-	procLdapFirstEntryW  = modWldap32.NewProc("ldap_first_entryW")
+	procLdapFirstEntry   = modWldap32.NewProc("ldap_first_entry")
 	procLdapGetValuesW   = modWldap32.NewProc("ldap_get_valuesW")
 	procLdapCountValuesW = modWldap32.NewProc("ldap_count_valuesW")
 	procLdapValueFreeW   = modWldap32.NewProc("ldap_value_freeW")
-	procLdapMsgFreeW     = modWldap32.NewProc("ldap_msgfreeW")
+	procLdapMsgFree      = modWldap32.NewProc("ldap_msgfree")
 )
 
 // from winldap.h
@@ -118,9 +118,9 @@ func LdapGetValue(
 	if int32(r1) != LDAP_SUCCESS {
 		return "", fmt.Errorf("ldap_search_sW failed: %d", r1)
 	}
-	defer procLdapMsgFreeW.Call(msg)
+	defer procLdapMsgFree.Call(msg)
 
-	entry, _, _ := procLdapFirstEntryW.Call(handle, msg)
+	entry, _, _ := procLdapFirstEntry.Call(handle, msg)
 	if entry == 0 {
 		return "", fmt.Errorf("no entries found")
 	}

